requirements.txt

```diff
@@ -0,0 +1,194 @@
+eo-learn-io==0.10.1
+eo-learn-mask==0.10.1
+eo-learn-ml-tools==0.10.1
+eo-learn-visualization==0.10.1
+Fiona==1.8.20
+flake8==4.0.1
+flatbuffers==2.0
+fs==2.4.14
+fs-s3fs==1.1.1
+gast==0.4.0
+geojson==2.5.0
+geopandas==0.8.2
+gitdb==4.0.9
+GitPython==3.1.24
+google-auth==1.35.0
+google-auth-oauthlib==0.4.6
+google-pasta==0.2.0
+gpytorch==1.6.0
+graphviz==0.18.2
+grpcio==1.42.0
+h5py==3.6.0
+idna==3.3
+imageio==2.11.1
+importlib-metadata==4.8.2
+importlib-resources==5.4.0
+iniconfig==1.1.1
+ipykernel==6.5.1
+ipython==7.29.0
+ipython-genutils==0.2.0
+ipywidgets==7.6.5
+jedi==0.18.1
+Jinja2==3.0.3
+jmespath==0.10.0
+joblib==1.1.0
+jsonschema==4.2.1
+jupyter==1.0.0
+jupyter-client==7.1.0
+jupyter-console==6.4.0
+jupyter-core==4.9.1
+jupyterlab-pygments==0.1.2
+jupyterlab-widgets==1.0.2
+keras==2.7.0
+Keras-Preprocessing==1.1.2
+kiwisolver==1.3.2
+libclang==12.0.0
+lightgbm==3.3.1
+llvmlite==0.37.0
+Markdown==3.3.6
+MarkupSafe==2.0.1
+marshmallow==3.14.1
+marshmallow-enum==1.5.1
+matplotlib==3.3.2
+matplotlib-inline==0.1.3
+mccabe==0.6.1
+mistune==0.8.4
+munch==2.5.0
+mypy-extensions==0.4.3
+nbclient==0.5.9
+nbconvert==6.3.0
+nbformat==5.1.3
+nest-asyncio==1.5.1
+networkx==2.6.3
+notebook==6.4.6
+numba==0.54.1
+numpy==1.19.5
+oauthlib==3.1.1
+opencv-contrib-python-headless==4.5.4.60
+opt-einsum==3.3.0
+packaging==21.3
+pandas==1.0.1
+pandocfilters==1.5.0
+parso==0.8.2
+pathspec==0.9.0
+pathtools==0.1.2
+pexpect==4.8.0
+pickleshare==0.7.5
+Pillow==8.4.0
+platformdirs==2.4.0
+pluggy==1.0.0
+prometheus-client==0.12.0
+promise==2.3
+prompt-toolkit==3.0.22
+protobuf==3.19.1
+psutil==5.8.0
+ptyprocess==0.7.0
+py==1.11.0
+pyasn1==0.4.8
+pyasn1-modules==0.2.8
+pycodestyle==2.8.0
+pycparser==2.21
+pydot==1.4.2
+pyflakes==2.4.0
+Pygments==2.10.0
+pyparsing==3.0.6
+pyproj==3.3.0
+pyrsistent==0.18.0
+pystac==1.2.0
+pytest==6.2.5
+python-dateutil==2.8.2
+pytz==2021.3
+PyWavelets==1.2.0
+PyYAML==6.0
+pyzmq==22.3.0
+qtconsole==5.2.0
+QtPy==1.11.2
+radiant-mlhub==0.3.1
+rasterio==1.1.8
+regex==2021.11.10
+requests==2.26.0
+requests-oauthlib==1.3.0
+rsa==4.7.2
+Rtree==0.9.7
+s3transfer==0.5.0
+scikit-image==0.18.3
+scikit-learn==1.0.1
+scipy==1.7.2
+seaborn==0.11.2
+Send2Trash==1.8.0
+sentinelhub==3.3.2
+sentry-sdk==1.5.0
+sh==1.14.2
+Shapely==1.8.0
+shortuuid==1.0.8
+six==1.16.0
+smmap==5.0.0
+snuggs==1.4.7
+subprocess32==3.5.4
+tensorboard==2.7.0
+tensorboard-data-server==0.6.1
+tensorboard-plugin-wit==1.8.0
+tensorflow-estimator==2.7.0
+tensorflow-io-gcs-filesystem==0.23.1
+termcolor==1.1.0
+terminado==0.12.1
+testpath==0.5.0
+threadpoolctl==3.0.0
+thunder-python==1.4.2
+thunder-registration==1.0.1
+tifffile==2021.11.2
+toml==0.10.2
+tomli==1.2.2
+torch==1.9.0
+torchvision==0.10.0
+tornado==6.1
+tqdm==4.62.3
+traitlets==5.1.1
+typing-extensions==4.0.0
+typing-inspect==0.7.1
+urllib3==1.26.7
+utm==0.7.0
+wandb==0.12.7
+wcwidth==0.2.5
+webencodings==0.5.1
+Werkzeug==2.0.2
+widgetsnbextension==3.5.2
+wrapt==1.13.3
+yaspin==2.1.0
+zipp==3.6.0
```

Comment:
isn't there already a requirements.txt in src?

Reply:
Oh I see it there.